scripts/get_bundle_test_path.py

```diff
@@ -0,0 +1,33 @@
+# Get bundle test path for specific release
+import re
+import sys
+import os
+
+
+def get_bundle_test_path_from_release() -> None:
+    if len(sys.argv) > 1:
+        release = sys.argv[1]
+        bundle_is_1_6 = re.search("^1.6/", release)
+        bundle_is_1_4 = re.search("^1.4/", release)
+        if bundle_is_1_4:
+            raise Exception(
+                f"No tests available for 1.4 release. Please proceed to manually publish to Charmhub if needed."
+            )
+        elif bundle_is_1_6:
+            bundle_test_path = "./tests-bundle/1.6/"
+        else:
+            bundle_test_path = "./tests-bundle/1.7/"
```

Comment:
Is our plan to add code here for next release or we will be using different solution?
For example, for 1.8 release we will have to add search for "^1.8" and add `elif` clause to this `if` statement. If this is what epxected, can we add a comment that outlines this process?
Eg.
```
# to add release to be processed by this script, please modify this script to . . .
```
Or alternatively create a dictionary that would map release to test bundle. This will be much easier to maintain and when adding new release the code will need to change only in one place.
Eg.
```
# add release to release tests mapping to this dictionary
RELEASE_TESTS = {
  "1.6/stable" : "./test-bundle/1.6",
  "1.7/stable" : "./test-bundle/1.7",
  "1.7/edge" : "./test-bundle/1.7",
}

release = "1.7/edge"

if release not in RELEASE_TESTS.keys():
    raise Exception(
      f"No tests available for {release}. Please proceed to manually publish to Charmhub if needed."
    )
bundle_test_path = RELEASE_TESTS[release]

```

Reply:
Great comment, I was actually pretty sure that there was a better way for this but hadn't figured this out.